exampleSite/content/post/post.1.en.md

```diff
@@ -2,7 +2,13 @@
 title = 'Primis eget imperdiet lorem'
 slug = 'post1'
 image = 'images/pic03.jpg'
-description = 'Example of post with missing date property'
+description = 'Example of post with missing date property butwith pic and video.'
 disableComments = true
 +++
-Example of post with missing date property.
\ No newline at end of file
+Example of post with missing date property but with pic and video.
+
+![img](/images/pic03.jpg)
+
+<div class="meta-media"><iframe src="//player.bilibili.com/player.html?aid=373841893&bvid=BV1Wo4y1R76n&cid=287681080&page=1" frameborder="no" scrolling="yes" allowfullscreen="allowfullscreen" high_quality="1" framespacing="1" class="video" > </iframe></div>
```

Comment:
Having to add HTML into the markdown files is not ideal. Fortunately I think Hugo provides a way of handling this with shortcodes:

https://gohugo.io/content-management/shortcodes

Reply:
I've also discovered a bug with this when removing the content beneath the video:

<img width="1197" alt="image" src="https://user-images.githubusercontent.com/15653985/162191704-2fb45797-7afe-4007-a00f-7bbec682675c.png">